pkg/kubernetes/styles/variables.less

```diff
@@ -4,4 +4,4 @@
 @sidebar-width-md: 145;
 @sidebar-width-xl: 185;
 
-@command-bg-color: #eee;
+@command-bg-color: var(--color-gray-2);
```

Comment:
If this LESS variable is used to compile another color, it could break. If it's used as-is, then that's great.